bundles/org.openhab.ui/web/src/assets/units.js

```diff
@@ -0,0 +1,164 @@
+//   default: default unit, to be set to override core default unit, not specific to SI or Imperial measurement system
+//   defaultSI: default unit in SI measurement system, to be set to override OH core default SI unit
+//   defaultUS: default unit in Imperial measurement system, to be set to override OH core default Imperial unit
+//   baseUnits: all supported base units that don't allow metric or binary prefixes
+//   baseUnitsMetric: metric base units, the full list of units will include all of these with all metric prefixes
+//   baseUnitsBinary: binary base units, the full list of units will include all of these with all binary prefixes
+// If nothing is defined for an allowed dimension, the UI will fall back on the OH core default unit in the configured
+// measurement system.
+// For dimensions defined, any of the fields can be ommitted. Logical defaults will be used.
+// Units from curated units lists will always be added to the full unit list constructed from baseUnits and prefixes. So it
+// is not necessary to explicitely add what is already in the curated units for the full units list. However, no prefixes
+// will be applied to these. If you want prefixes to be applied, you should add them in the respective baseUnits Array as well.
+
+export const Units = [{
+  dimension: 'Acceleration',
+  baseUnits: ['gₙ'],
+  baseUnitsSI: ['m/s²']
+}, {
+  dimension: 'AmountOfSubstance',
+  baseUnits: ['°dH'],
+  baseUnitsSI: ['mol']
+}, {
+  dimension: 'Angle',
+  units: ['°', '\'', '"', 'rad']
+}, {
+  dimension: 'Area',
+  unitsSI: ['m²', 'km²', 'ha'],
+  unitsUS: ['ft²', 'mi²'],
+  baseUnits: ['ca', 'a', 'in²', 'ac'],
+  baseUnitsMetric: ['m²']
+}, {
+  dimension: 'DataAmount',
+  units: ['bit', 'B', 'kB', 'kiB', 'MB', 'MiB', 'GB', 'GiB', 'TB', 'TiB'],
+  baseUnitsMetric: ['bit', 'B', 'o'],
+  baseUnitsBinary: ['bit', 'B', 'o']
+}, {
+  dimension: 'DataTransferRate',
+  units: ['bit/s', 'kbit/s', 'Mbit/s', 'GBit/s'],
+  baseUnitsMetric: ['bit/s'],
+  baseUnitsBinary: ['bit/s']
+}, {
+  dimension: 'Density',
+  units: ['g/l', 'g/m³', 'kg/m³'],
+  baseUnits: ['lb/in³'],
+  baseUnitsMetric: ['g/m³', 'g/mm³', 'g/cm³', 'g/dm³', 'g/ml', 'g/cl', 'g/dl', 'g/l']
+}, {
+  dimension: 'Dimensionless',
+  units: ['one', '%', 'dB', 'ppm', 'ppb'],
+  default: '%'
+}, {
+  dimension: 'ElectricCapcitance',
+  baseUnitsMetric: ['F']
+}, {
+  dimension: 'ElectricCharge',
+  units: ['Ah', 'C'],
+  baseUnitsMetric: ['C']
+}, {
+  dimension: 'ElectricConductance',
+  baseUnitsMetric: ['S']
+}, {
+  dimension: 'ElectricConductivity',
+  baseUnitsMetric: ['S/m']
+}, {
+  dimension: 'ElectricCurrent',
+  baseUnitsMetric: ['A']
+}, {
+  dimension: 'ElectricInductance',
+  baseUnitsMetric: ['H']
+}, {
+  dimension: 'ElectricPotential',
+  baseUnitsMetric: ['V']
+}, {
+  dimension: 'ElectricResistance',
+  baseUnitsMetric: ['Ω']
+}, {
+  dimension: 'Energy',
+  units: ['kWh', 'Wh', 'VAh', 'varh', 'J', 'kJ', 'cal', 'kcal'],
```

Comment:
IMHO these curated entries should be `kVAh` and `kvarh` -- with `k` prefix rather than without; by analogy to line #117 below..

Reply:
fixed